docs/docs/reference/language-versions/binary-compatibility.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: doc-page
+title: "Binary Compatibility"
+---
+
+Thanks to TASTy files, which are produced during compilation of Scala 3 sources, unlike Scala 2, Scala 3 is backward binary compatible between different minor versions (i.e. binary artifacts produced with Scala 3.x can be consumed by Scala 3.y programs as long as x <= y).
```

Comment:
The presence of tasty files by themselves does not guarantee backward binary compatibility since binary compat and tasty compat are orthogonal (it's possible to break one will preserving the other) so I wouldn't say that binary compat is preserved "thanks to tasty files".